addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -228,3 +228,36 @@ img.o_forum_avatar_big {
 .website_forum {
     margin-bottom: $spacer;
 }
+
+.forums main {
```

Comment:
You are not supposed to style the `main` element, if you enable the "boxed" layout with the customize dialog, this will appear broken. I suppose you may target the `#wrap` element instead. In a general way, we try not to target elements by tag name but by class names.